lib/template.html

```diff
@@ -98,11 +102,27 @@
 {% endif %}
 
 {% if page.image %}
-  {% assign seo_page_image = page.image.path | default: page.image.facebook | default: page.image %}
-  {% unless seo_page_image contains "://" %}
-    {% assign seo_page_image = seo_page_image | absolute_url %}
+  {% assign seo_page_image_default = page.image.default.path | default: page.image.default | default: page.image.path | default: page.image %}
```

Comment:
I don't mind stacked defaults. It let's me see at a glance what the precedence ordering is :+1: